docs/src/pages/guides/server-rendering/server-rendering-ja.md

```diff
@@ -2,18 +2,18 @@
 
 <p class="description">サーバーサイドレンダリングの最も一般的な使用例は、ユーザー（または検索エンジンのクローラー）が最初にアプリを要求した際に、最初に提示される画面を描画するのに使われます。</p>
 
-サーバはリクエストを受け付けると、必要なコンポーネントをHTMLフォーマットの文字列として書き出し、クライアント側に送り返します。 そして、初回リクエスト以降のレンダリングはクライアント側で行います。 そして、初回リクエスト以降のレンダリングはクライアント側で行います。
```

Comment:
It would be better to leave the translation untouched. We auto-sync them.

Reply:
Actually I never changed anything other than the imports. I am also surprised that translations got changed.